lib/deimos/active_record/message_consumption.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Deimos
+  module ActiveRecord
+    # Consumption methods
+    module MessageConsumption
```

Comment:
Starting to get confused with all these modules and how they relate to the consumers now - can you cleanly state what each module is responsible for? If not, maybe the modules should be split up differently?